setup.py

```diff
@@ -43,19 +43,21 @@
         # The following is for both SS and Envision
         "cloudpickle>=1.3.0,<1.4.0",
         # The following are for /envision
-        "tornado>=6.1",
+        "tornado>=5.1.1",
         "websocket-client>=1.2.1",
         # The following is used for imitation learning and envision
         "ijson>=3.1.4",
         # The following are for the /smarts/algorithms
-        "matplotlib>=3.4.3",
+        "matplotlib>=3.2.2",
         # The following are for /smarts/zoo and remote agents
-        "grpcio==1.32.0",
-        "protobuf==3.20.1",
-        "PyYAML>=6.0",
+        "protobuf>=3.17.3",
+        "PyYAML>=3.13",
         "twisted>=21.7.0",
     ],
     extras_require={
+        "opendrive": [
+            "opendrive2lanelet>=1.2.1",
+        ],
```

Comment:
The extra option `opendrive` seems to be repeated twice, and thus this copy can be deleted.